src/api/AppActivity.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * @flow
+ * @file Helper for the box App Activity API
+ * @author Box
+ */
+import MarkerBasedAPI from './MarkerBasedAPI';
+import { ERROR_CODE_DELETE_APP_ACTIVITY, HTTP_STATUS_CODE_NOT_FOUND } from '../constants';
+import { APP_ACTIVITY_FIELDS_TO_FETCH } from '../utils/fields';
+
+class AppActivity extends MarkerBasedAPI {
+    /** @property {BoxItemPermission} - Placeholder permissions object to determine if app activity can be deleted */
+    permissions: BoxItemPermission = {};
+
+    /**
+     * Map an entry from the AppActivity API to an AppActivityItem.
+     * occurred_at -> created_at
+     * Adds permissions to item
+     *
+     * @param {Object} item - A single entry in the AppActivity API entries list
+     *
+     * @return {AppActivityItem}
+     */
+    mapAppActivityItem = (item: Object): AppActivityItem => {
```

Comment:
Can we avoid `Object` here?

Reply:
I'll whip up another type for the API response shape